db/1.8/modules/ROOT/pages/database-connector-connection.adoc

```diff
@@ -0,0 +1,393 @@
+. Click *Test Connection* to confirm that Mule can connect to the database.
+. Click *OK*.
+
+The following screenshot shows the *Derby Connection* configuration in Studio:
+
+.Derby Connection configuration
+image::database-derby-connection.png[Derby Connection configuration]
+
+In the XML editor, the configuration looks like this:
+[source,xml,linenums]
+----
+<db:config name="Database_Config" >
+		<db:derby-connection database="derbyDB" />
+	</db:config>
+----
+
+== Configure a Generic Connection
+
+Use the *Generic Connection* type when you want to connect to:
+
+* A database for which MuleSoft does not provide a specific connection type.
+* A supported database and you are using custom options that are not included in the connection types.
+
+The following example shows how to connect to an H2 database in Anypoint Studio:
+
+. In Studio, navigate to the *Global Elements* view.
+. Click *Create* to open the *Choose Global Type* view.
+. In the *Filter* field write `database`, select *Database Config* and click *OK*.
+. In the *Database Config* window, for *Connection* select *Generic Connection*.
+. Click *Configure* to add the required JDBC driver and select either: +
++
+* *Use local file* +
+Install the library using a local file.
+* *Add Maven dependency* +
+Install a Maven dependency to add to the project.
++
+. Set the connection *URL* to `jdbc:h2:file:/tmp/datasenseDBand` and *Driver class name* to `org.h2.Driver`.
+. Click *Test Connection* to validate the connection with the database.
+. Click *OK*.
+
+The following screenshot shows the *Generic Connection* configuration in Studio:
+
+.Generic Connection configuration
+image::database-generic-connection.png[Generic Connection configuration]
+
+In the XML editor, the configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:config name="Database_Config">
+    <db:generic-connection driverClassName="org.h2.Driver"
+    url="jdbc:h2:file:/tmp/datasenseDB"/>
+</db:config>
+----
+
+== Configure a Microsoft SQL Server Connection
+
+Use the *Microsoft SQL Server Connection* type when you want to connect to a Microsoft SQL Server database.
+
+The following example shows how to configure a *Microsoft SQL Server Connection* in Anypoint Studio:
+
+. In Studio, navigate to the *Global Elements* view.
+. Click *Create* to open the *Choose Global Type* view.
+. In the *Filter* field write `database`, select *Database Config* and click *OK*.
+. In the *Database Config* window, for *Connection* select *Microsoft SQL Server Connection*.
+. Click *Configure* to add the required Microsoft SQL Server driver and select either: +
++
+* *Add recommended library* +
+Install the recommended library.
+* *Use local file* +
+Install the library using a local file.
+* *Add Maven dependency* +
+Install a Maven dependency to add to the project.
++
+. Set *Host* to `0.0.0.0`, *Port* to `553`, *User* to `Max`, *Password* to `password` and *Database name* to `microsoftDB`.
```

Comment:
GLOBAL style is to use a comma before the final item in a list ("the Oxford comma"):
```suggestion
. Set *Host* to `0.0.0.0`, *Port* to `553`, *User* to `Max`, *Password* to `password`, and *Database name* to `microsoftDB`.
```